pkg/connect/pubsub/proxy.go

```diff
@@ -0,0 +1,295 @@
+package pubsub
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/google/uuid"
+	connect_sdk "github.com/inngest/inngestgo/connect"
+	"github.com/oklog/ulid/v2"
+	"github.com/redis/rueidis"
+	"sync"
+	"time"
+)
+
+type ProxyResponse struct {
+	Status string
+
+	SdkResponse *connect_sdk.SdkResponse
```

Comment:
the `SdkResponse` is better to be defined in this repo, and have the SDK reference it instead.
Otherwise, Go SDK becomes the source of truth, which will be weird.

Reply:
Good point, done